File: 01FlutterApps/middle_layer_springboot/knpiano_java/src/main/java/com/liu/springboot04web/bean/Kn03D003StubnkBean.java
package com.liu.springboot04web.bean;


import java.util.Date;

public class Kn03D003StubnkBean implements KnPianoBean {
    private String stuId;
    private String bankId;
    private String stuName;
    private String bankName;
    private Integer delFlg;
    private Date createDate;
    private Date updateDate;
    public String getStuId() {
        return stuId;
    }
    public void setStuId(String stuId) {
        this.stuId = stuId;
    }
    public String getBankId() {
        return bankId;
    }
    public void setBankId(String bankId) {
        this.bankId = bankId;
    }
    public String getStuName() {
        return stuName;
    }
    public void setStuName(String stuName) {
        this.stuName = stuName;
    }
    public String getBankName() {
        return bankName;
    }
    public void setBankName(String bankName) {
        this.bankName = bankName;
    }
    public Integer getDelFlg() {
        return delFlg;
    }
    public void setDelFlg(Integer delFlg) {
        this.delFlg = delFlg;
    }
    public Date getCreateDate() {
        return createDate;
    }
    public void setCreateDate(Date createDate) {
        this.createDate = createDate;
    }
    public Date getUpdateDate() {
        return updateDate;
    }
    public void setUpdateDate(Date updateDate) {
        this.updateDate = updateDate;
    }
}

File: 01FlutterApps/front_dart_flutter/lib/03StuDocMngmnt/3bankBasic/Kn03D003StubnkBean.dart
class Kn03D003StubnkBean {
  final String bankId;
  final String bankName;
  final String stuId;
  final String stuName;
  final int delFlg;

  Kn03D003StubnkBean({
    required this.bankId, 
    required this.bankName, 
    required this.stuId, 
    required this.stuName, 
    required this.delFlg,
    });

  factory Kn03D003StubnkBean.fromJson(Map<String, dynamic> json) {
    return Kn03D003StubnkBean(
      bankId: json['bankId'],
      bankName: json['bankName'],
      stuId: json['stuId'],
      stuName: json['stuName'],
      delFlg: json['delFlg'] ?? 0,
    );
  }
}
